k8s/scripts/provision.sh

```diff
@@ -90,14 +90,31 @@ done
 
 kubectl --kubeconfig=/etc/kubernetes/admin.conf get pods -n kube-system
 
-kubeadm reset
+export k8s11=false
+if [[ $version =~ \.([0-9]+) ]] && [[ ${BASH_REMATCH[1]} -ge "11" ]]; then
```

Comment:
Can you use something more simple, for examlpe:
```
if [[ $version =~ .*-1.11..* ]]; then
  reset_command="kubeadm reset --force"
  admissionflag="enable-admission-plugins"
else
  ...
fi
```

Reply:
My check will also work for future versions, the '-ge' means greater or equal. I'd prefer that one (and hope that 1.12 won't need other changes in this area ;) ).
But I can group the 2 ifs together like you did, that's a bit nicer indeed.